<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="RunButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FindButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>50</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="StackedWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>801</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QWidget" name="Input">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>0</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>输入函数的信息</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>函数名称</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>多项式次数</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Xmax</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Xmin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>A真实值    （未使用的a请置为0）</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>320</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>a1=</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>320</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>a2=</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>320</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>a3=</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>360</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>a4=</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>360</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>a5=</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>360</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>a6=</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>360</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>a7=</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>320</y>
       <width>72</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>a0=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameline">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>80</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>130</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="nspinBox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="XmaxdoubleSpinBox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>170</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="XmindoubleSpinBox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="a0SpinBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>320</y>
       <width>70</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="a1SpinBox">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>320</y>
       <width>70</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="a2SpinBox">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>320</y>
       <width>70</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="a3SpinBox">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>320</y>
       <width>70</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="a4SpinBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>360</y>
       <width>70</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="a5SpinBox">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>360</y>
       <width>70</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="a6SpinBox">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>360</y>
       <width>70</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="a7SpinBox">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>360</y>
       <width>70</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Show">
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>显示数据</string>
     </property>
    </widget>
    <widget class="QTableView" name="a_tableView">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>291</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="a__tableView">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>301</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>320</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>真实的a</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>330</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>拟合后的a'</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>380</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>显示</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>380</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Find">
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>0</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查询函数</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>390</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>函数次数</string>
     </property>
    </widget>
    <widget class="QPushButton" name="FindnameButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>390</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>查找</string>
     </property>
    </widget>
    <widget class="QTableView" name="a__tableView_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>50</y>
       <width>341</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="a_tableView_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>311</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>320</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>真实的a</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>330</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>拟合后的a'</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>390</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="Cpwd">
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>原密码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>新密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="oldpwdlineEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>70</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="newpwdlineEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>140</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="submitButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>280</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>280</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="ShowButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>显示</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InputButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>输入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EvaluateButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>评估</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CpwdButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>50</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
